gitrevise/odb.py

```diff
@@ -217,28 +220,27 @@ def git(
         self,
         *cmd: str,
         cwd: Optional[Path] = None,
+        env: Dict[str, str] = None,
         stdin: Optional[bytes] = None,
+        stdout: _FILE = PIPE,
```

Comment:
OK. I guess the added consistency is because now input-related parameters come before all output-related ones.